src/oe-thesaurus-tree.ts

```diff
@@ -80,7 +80,18 @@ export class OeThesaurusTree {
   public updateValue(id: number) {
     this.service.getConceptById(this.config.type, id).then((data) => {
       if (data) {
-        this.value = new Member(data.id, data.label, data.type, data.uri);
+        let selectedLabel = null;
```

Comment:
Misschien herschrijven naar dit? 

![image](https://user-images.githubusercontent.com/9083121/119114198-38d5ee80-ba26-11eb-8b88-32b3a79c9a75.png)

Reply:
Dan klopt de flow niet meer. Die data.label is het default label dat ingesteld is. In uw tweede shorthand if gaat die length groter dan 0 zijn, dan ga je het eerste element nemen maar dat is een string (data.label).

Reply:
```
{
    "id": 72,
    "type": "concept",
    "uri": "https://id.erfgoed.net/thesauri/soorten/72",
    "label": "palmboom",
    "labels": [
        {
            "label": "Buxus",
            "type": "prefLabel",
            "language": "la"
        },
        {
            "label": "palmboom",
            "type": "altLabel",
            "language": "nl-BE"
        }
    ]
}
```

Als voorbeeld

Reply:
![image](https://user-images.githubusercontent.com/9083121/119128533-11871d80-ba36-11eb-8c52-0eb1584729d6.png)

Dan kunt ge het volgens mij zo doen, dan zet ge uw default waarden op de else waarden voordien en als er dan een config.language is dan kan je nog zien welke het moet worden